docs/profiles.md

```diff
@@ -81,20 +81,22 @@ $ inspec check examples/profile
 
 # Platform Support
 
-Use the `supports` setting in the `inspec.yml` file to specify one (or more) platforms for which a profile is targeting. The list of supported platforms may contain simple names, names and versions, or detailed flags, and may be combined arbitrarily. For example, to target anything running Debian Linux:
+Use the `supports` setting in the `inspec.yml` file to specify one (or more) platforms for which a profile is targeting. The list of supported platforms may contain simple names, names and versions, or detailed flags, and may be combined arbitrarily. 
```

Comment:
Could we have an explicit list of the keys that are allowed here? ` may contain simple names, names and versions, or detailed flags` does not tell what is allowed and what isn't.  Examples are good, but don't tell you the complete picture.